src/custom/state/orders/utils.ts

```diff
@@ -208,7 +208,7 @@ export function getOrderExecutedAmounts(order: OrderMetaData): {
  * @param order
  * @param price
  */
-export function isOrderUnfillable(order: Order, price: Required<PriceInformation>): boolean {
+export function isOrderUnfillable(order: Order, price: Required<Omit<PriceInformation, 'quoteId'>>): boolean {
```

Comment:
Why do we explicitly omit it? 

I mean, we don't need it probably right now, but I the quoteId will be there, right? and we could in principle even report what two quotes are in conflict (quote of order, vs quote of the last price)

Reply:
thanks for clarifying. Then u can leave it as it is